### ISSUE TITLE
Delete `devolop` branch

### ISSUE THREAD

**Issue by dilyanpalauzov** [agendav/agendav#302]
Currently agendav on git  has as branches `develop` and `development`, where only the latter has seen recently seen updates.  Likewise it has the branches `master` and `main`, where only `main` has recent updates.

Please delete the branches `develop` and `master`, as these lead only to confusion.

**Comment by pixelbrackets** [comment]
You are absolutely right. This has to wait until the next release however, since the latest release still points to `develop` instead of `development` in many places (this issue was fixed only recently, due to missing access rights).

**Comment by pixelbrackets** [comment]
All links fixed in latest release. Existing PRs need to be resolved however, since they are closed automatically upon deleting the connected branch.

**Comment by pixelbrackets** [comment]
I finally merged/closed all remining PRs - now the develop branch was safe to remove
